docs/Release_Guide/release_steps/met/update_tar_files_link_official.rst

```diff
@@ -0,0 +1,8 @@
+Update tar_files Link
+^^^^^^^^^^^^^^^^^^^^^
+
+On mohawk, in /d2/www/dtcenter/dfiles/code/METplus/MET/installation/, update
+the link for tar_files.latest.tgz, to point to the latest version of the
+tar_files.met-base-vX.Y.tgz file.
```

Comment:
```suggestion
tar_files.met-base-vX.Y.tgz file and confirm the result at https://dtcenter.ucar.edu/dfiles/code/METplus/MET/installation/.
```

Reply:
@JohnHalleyGotway when you say "and confirm the result at https://dtcenter.ucar.edu/dfiles/code/METplus/MET/installation/."  Do you mean by checking the size of the file?